pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1815,6 +1815,29 @@ type APIServerConfig struct {
 	BearerToken string `json:"bearerToken,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=v1;V1;v2;V2
+type AlertmanagerAPIVersion string
+
+const (
+	AlertmanagerAPIVersion1       = AlertmanagerAPIVersion("V1")
+	AlertmanagerAPIVersion2       = AlertmanagerAPIVersion("V2")
+	AlertmanagerAPIVersionUnKnown = AlertmanagerAPIVersion("")
+)
+
+func (av *AlertmanagerAPIVersion) ToUpper() AlertmanagerAPIVersion {
```

Comment:
the comparison is so simple that I wouldn't add this to the pkg/apis/monitoring package.

Reply:
Remove it done.